packages/frontend/app/components/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,161 @@
+import { Dialog, Transition } from '@headlessui/react'
+import {
+  forwardRef,
+  Fragment,
+  useImperativeHandle,
+  useRef,
+  useState
+} from 'react'
+import { ExclamationTriangle, X } from './icons'
```

Comment:
```suggestion
import { ExclamationTriangle, X as XIcon } from './icons'
```
Maybe a bit more straightfoward